### PR TITLE
Add config lib to ISIS base feature profile

### PR DESCRIPTION
**Pull request by sthesayi**
Also fixed couple of errors and dependencies in the feature textproto. The config lib is for the entirety of ISIS base feature profile and is broken down into multiple files per sub tree. See BGP base feature profile for a similar pattern.

### CODE REVIEW

File: feature/isis/isis.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+// WithLSPRefreshInterval sets the lsp-refresh-interval for ISIS global.
+func (i *ISIS) WithLSPRefreshInterval(interval time.Duration) *ISIS {
+	i.oc.GetOrCreateIsis().GetOrCreateGlobal().GetOrCreateTimers().LspRefreshInterval = ygot.Uint16(uint16(interval.Seconds()))
+	return i
+}
+
+// WithSPFFirstInterval sets the spf-first-interval for ISIS global.
+func (i *ISIS) WithSPFFirstInterval(interval time.Duration) *ISIS {
+	i.oc.GetOrCreateIsis().GetOrCreateGlobal().GetOrCreateTimers().GetOrCreateSpf().SpfFirstInterval = ygot.Uint64(uint64(interval.Seconds()))
```
Comment (bstoll) [inline feature/isis/isis.go:83]: Should this be interval.Milliseconds() - [ref](https://github.com/openconfig/public/blob/f3349778e3cb3b0987f3ec18acea329ea4bf30e6/release/models/isis/openconfig-isis.yang#L729) ?

File: feature/isis/isis.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	return i
+}
+
+// WithSPFHoldInterval sets the spf-hold-interval for ISIS global.
+func (i *ISIS) WithSPFHoldInterval(interval time.Duration) *ISIS {
+	i.oc.GetOrCreateIsis().GetOrCreateGlobal().GetOrCreateTimers().GetOrCreateSpf().SpfHoldInterval = ygot.Uint64(uint64(interval.Seconds()))
```
Comment (bstoll) [inline feature/isis/isis.go:89]: Should this be interval.Milliseconds() - [ref](https://github.com/openconfig/public/blob/f3349778e3cb3b0987f3ec18acea329ea4bf30e6/release/models/isis/openconfig-isis.yang#L721)?

**Review by bstoll — APPROVED** [review]

File: feature/isis/interface.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+func (i *Interface) WithCSNPInterval(interval time.Duration) *Interface {
+	toc := i.oc.GetOrCreateTimers()
+	toc.CsnpInterval = ygot.Uint16(uint16(interval.Seconds()))
+	return i
+}
+
+// WithLSPPacingInterval sets the lsp-pacing-interval on the interface ISIS.
+func (i *Interface) WithLSPPacingInterval(interval time.Duration) *Interface {
+	toc := i.oc.GetOrCreateTimers()
+	toc.LspPacingInterval = ygot.Uint64(uint64(interval.Seconds()))
```
[the remark this answers is not in the document]
Comment (sthesayi) [reply feature/isis/interface.go]: Nice catch!!

File: feature/isis/isis.go (hunk 1 above)
Comment (sthesayi) [reply feature/isis/isis.go]: done

File: feature/isis/isis.go (hunk 2 above)
Comment (sthesayi) [reply feature/isis/isis.go]: Done

**Review by bstoll — APPROVED** [review]

**Pull request merged**